dbus/dbus_libgio.go.h

```diff
@@ -0,0 +1,92 @@
+// exported by golang, see dbus_libgio.go
+GVariant *
+handle_method_call_callback(gchar *objectPath,
+                            gchar *interfaceName,
+                            gchar *methodName);
+
+// convert an unsafe pointer to a GDBusConnection structure
+static GDBusConnection*
+to_gdbusconnection(void* ptr)
+{
+	return (GDBusConnection*)ptr;
+}
+
+// convert an unsafe pointer to a GMainLoop structure
+static GMainLoop*
+to_gmainloop(void* ptr)
+{
+	return (GMainLoop*)ptr;
+}
+
+// create a new GVariant from a string value
+static GVariant *g_variant_new_from_string(gchar *value)
+{
+    return g_variant_new("(s)", value);
```

Comment:
Inconsistent indentation in several places in this file. I would argue we should use tabs, as in the Go code.

Reply:
I used spaces and tried to make it consistent, don't know if I can use some sort-of formatter.
I also added an editorconfig file to enforce it.
What do you think?